Extensions/3D/JsExtension.js

```diff
@@ -2746,13 +2748,24 @@ module.exports = {
         );
         threeObject.updateMatrixWorld(true);
         const boundingBox = new THREE.Box3().setFromObject(threeObject);
+        const shouldKeepModelOrigin = !this._originPoint;
+        if (shouldKeepModelOrigin) {
+          // Keep the origin as part of the model.
+          // For instance, a model can be 1 face of a cube and we want to keep the
+          // inside as part of the object even if it's just void.
+          // It also avoids to have the origin outside of the object box.
```

Comment:
> It also avoids to have the origin outside of the object box.

i'm not sure what this means exactly?

Reply:
Currently, if you keep the model origin as the object origin and the model origin is at -1000 ; -1000, when the object position is at 0 ; 0, the collision mask will be at around 1000 ; 1000 instead of from 0 to around 1000 on both axis.